sandy/fy.py

```diff
@@ -441,20 +442,20 @@ def apply_bmatrix(self, zam, e, decay_data, keep_fy_index=False):
             Energy to which calculations are to be applied.
         decay_data : `sandy.DecayData`
             Radioactive nuclide data for several isotopes.
-        keep_fy_index=False : `bool`, optional
+        keep_fy_index: `bool`, optional, default is `False`
```

Comment:
It is weird to call it `keep_fy_index` since the fy dataframe does not have a real index. To be changed in a new PR